index.html

```diff
@@ -35,6 +35,173 @@ <h1 class="author">I’m Asif Amin Glad to see you!</h1>
+          </ul>
+          <button type="button" class="default_button">See More</button>
+        </div>
+      </div>
+
+      <!-- work card 4-->
+      <div class="card">
+        <div class="preview">
+          <img src="./assets/snapshoot-portfolio4.png" alt="Project 4" />
+        </div>
+        <div class="details">
+          <h2 class="title">Multi-Post Stories</h2>
+          <ul class="info">
+            <li class="info_item">CANOPY</li>
+            <li class="info_item">Back End Dev</li>
+            <li class="info_item">2015</li>
+          </ul>
+          <p class="details_content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="catagories">
+            <li>Html</li>
+            <li>Css</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="default_button">See More</button>
+        </div>
+      </div>
+    </section>
+    <!-- end of work section-->
+
+    <section class="about">
+      <h2 class="headline_title">About Myself</h2>
+      <p class="about_details">
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don’t hestiate to
+        contact me.
+      </p>
+      <a href="#" class="connect_button">let's connect</a>
+
+        <div class="social">
+          <ul>
+            <li><a href="#"><img src="./assets/Vector.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="./assets/Linkedin icon.svg" alt="Linkedin"></a></li>
+            <li><a href="#"><img src="./assets/Vector-2.svg" alt="m-icon"></a></li>
+            <li><a href="#"><img src="./assets/Vector-1.svg" alt="github"></a></li>
+            <li><a href="#"><img src="./assets/Vector-3.svg" alt="victory"></a></li>
+          </ul>
+        </div>
+      <div class="action">
+        <button type="button" class="default_button">Get my resume</button>
+      </div>
+      <ul class="about_ul">
+        <h3 class="about_sub_title">
+          Languages
+          <img
+            class="down_arrow_icon"
+            src="./assets/down-arrow-icon.png"
+            alt="down"
+          />
+        </h3>
+        <li class="ul_item">
+          <img src="./assets/js-icon.png" alt="Javascript" /><span
+            >Javascript</span
+          >
+        </li>
+        <li class="ul_item">
+          <img src="./assets/html-icon.png" alt="HTML" /><span>HTML</span>
+        </li>
+        <li class="ul_item">
+          <img src="./assets/css-icon.png" alt="CSS" /><span>CSS</span>
+        </li>
+      </ul>
+      <ul class="about_ul">
+        <h3 class="about_sub_title">
+          Frameworks
+          <img
+            class="arrow_icon"
+            src="./assets/right-arrow-icon.png"
+            alt="down"
+          />
+        </h3>
```

Comment:
:red_circle: [OPTIONAL] Only li tags should be direct children of ul tag. Kindly wrap h3 inside a li tag